module/PowerShellEditorServices.VSCode/PowerShellEditorServices.VSCode.psd1

```diff
@@ -102,7 +102,7 @@ PrivateData = @{
         # Tags = @()
 
         # A URL to the license for this module.
-        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/master/LICENSE'
+            LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```

Comment:
```suggestion
        LicenseUri = 'https://github.com/PowerShell/PowerShellEditorServices/blob/main/LICENSE'
```

Think the indent here was unintentional

Reply:
Thanks, super weird, don't know why Code did that. All I did was click the replace button 🤦